src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -394,11 +368,13 @@ class SchedulerActions(
     (instances.allSpecIdsWithInstances -- runSpecIds).foreach { unknownId =>
       logger.warn(
         s"RunSpec $unknownId exists in InstanceTracker, but not store. " +
-          "The run spec was likely terminated. Will now expunge."
+          "The run spec was likely terminated. Will now decommission."
       )
       val orphanedInstances = instances.specInstances(unknownId)
```

Comment:
I'm not sure you need to fetch the instances as we only need the ids.

Reply:
how else would you get instance ids from appId (unknownId is PathId, not instanceId)

Reply:
Ah, `instances` is `InstancesBySpec`. Then it does not matter.